wikimedia/wikipedia-ios
# Image recommendations - Improve error handling

**Phabricator:** https://phabricator.wikimedia.org/T358928#9761814

### Notes
Adding internet connection error banners on the "Add image details" and "Preview" views.

### Test Steps
1. Go to image recommendations
2. Before tapping "yes", turn on Airplane mode.
3. Confirm "No internet connection" banner appears after tapping Yes.
4. Tap Next
5. Confirm "No internet connection" banner appears on Preview.
